gradle/libs.versions.toml

```diff
@@ -1,21 +1,25 @@
 [versions]
-managed-liquibase = "4.17.2"
+managed-liquibase = "4.18.0"
```

Comment:
if this is a breaking change, back it out and consider working it separately from the renovate PR

Reply:
The previous version doesn't work either, so I reverted and then went further back to the version in the last release (v5.5.0)